python/friesian/example/multi_task/run_multi_task.py

```diff
@@ -0,0 +1,250 @@
+#
+# Copyright 2016 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import math
+from time import time
+from argparse import ArgumentParser, ArgumentError
+from keras.callbacks import EarlyStopping
+
+from bigdl.orca import init_orca_context, stop_orca_context
+from bigdl.orca.learn.tf2.estimator import Estimator
+from bigdl.friesian.feature import FeatureTable
+
+from deepctr.feature_column import SparseFeat, DenseFeat
+from deepctr.models import MMOE, PLE
+
+
+def build_model(model_type, sparse_features, dense_features, feature_max_idx):
+    sparse_feature_columns = [SparseFeat(feat, feature_max_idx[feat],
+                                         embedding_dim='auto') for feat in sparse_features]
```

Comment:
what's the behavior of embedding_dim=auto?

Reply:
if set auto: embedding_dim = 6 * int(pow(vocabulary_size, 0.25))